packages/plugin-legacy/src/index.ts

```diff
@@ -183,6 +197,15 @@ function viteLegacyPlugin(options: Options = {}): Plugin[] {
               : legacyEnvVarMarker
         }
       }
+    },
+    configResolved(config) {
+      if (overriddenBuildTarget) {
+        config.logger.warn(
+          colors.yellow(
+            `plugin-legacy overrode 'build.target'. You should pass 'targets' option to legacy browser instead.`
```

Comment:
```suggestion
            `plugin-legacy overrode 'build.target'. You should pass 'targets' as an option to this plugin with the list of legacy browsers to support instead.`
```

Reply:
Oh, this was a very strange sentence. 😅Thanks!